docs/embed/use-case/web-app.md

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
This is not the use case for embedding. This doc tells the user how to create a Wasm image.

Reply:
You can't solve the problems without changing the content.

Reply:
Please remove this chapter.